code/game/objects/effects/effects.dm

```diff
@@ -4,9 +4,10 @@
 /obj/effect
 	icon = 'icons/effects/effects.dmi'
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
+	attackable = FALSE
 
 /obj/effect/attackby(obj/item/I, mob/living/user, params)
-	return
+	return ..()
```

Comment:
This proc doesn't need to exist

Reply:
i hoped to prevent a future coder's frustration

Reply:
If someone adds an attackby component on an effect it won't work

Reply:
Why is that?

Reply:
because the signal is sent by atom/attackby(), and if /obj/effect/attackby does not call ..() it will not not send the signal

Reply:
But now it IS calling ..() making the entire definition here useless

Reply:
oh i'm dumb, sorry